artifacts/kindClusterConfig/general-config.yaml

```diff
@@ -4,8 +4,6 @@ networking:
   disableDefaultCNI: {{disable_cni}}
   podSubnet: {{pod_cidr}}
   serviceSubnet: {{service_cidr}}
-featureGates:
```

Comment:
This feature is enabeled to be true after 1.19, so it's ok

Reply:
good!

Reply:
That means we don't need this flag, right? Since our E2E are started to v1.21.

Reply:
Yes, we don't need it, it's enabled by default